src/login/logind-dbus.c

```diff
@@ -1186,6 +1186,13 @@ static int method_release_session(sd_bus_message *message, void *userdata, sd_bu
         if (r < 0)
                 return r;
 
+        r = get_sender_session(m, message, /* consult_display= */ false, error, &sender_session);
+        if (r < 0)
+                return r;
+
+        if (session != sender_session)
+                return sd_bus_error_set(error, BUS_ERROR_NOT_IN_CONTROL, "You are not in control of this session");
```

Comment:
hmm, please define a new error for this. the NotInControl error is really about session controllers, but this here really is different. Please add a new error.

Or just return SD_BUS_ERROR_ACCESS_DENIED or so.